### PR TITLE
Add advice on recording meetings

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
As discussed in the coordination meeting of 2021-05-27

### CODE REVIEW

File: what_are_WGs.md [hunk 1]
```diff
@@ -82,3 +82,21 @@ When the HSF organises [workshops](https://indico.cern.ch/category/7971/)
 working group conveners can give input to the organising team. If there
 will be a session dedicated to the activities of any working group then
 conveners will be asked to co-organise it.
+
+### Recording Meetings and Minutes
+
+In general recording *is appropriate* for HSF meetings, which are there to
+foster discussion and collaboration so *we do recommend it*. There are a lot of
+meetings and we have a worldwide community, so not everyone who is interested
+can join the meeting and a recording can help people to catch-up.
```
Comment (eduardo-rodrigues) [inline what_are_WGs.md:91]:
```suggestion
can join a meeting and recordings can help people to catch-up.
```

File: what_are_WGs.md [hunk 2]
```diff
@@ -82,3 +82,21 @@ When the HSF organises [workshops](https://indico.cern.ch/category/7971/)
...
+
+However, you need to make make it very clear with the speakers and to participants that:
```
Comment (eduardo-rodrigues) [inline what_are_WGs.md:93]:
```suggestion
However, you need to make it very clear with the speakers and to participants that:
```

File: what_are_WGs.md [hunk 3]
```diff
@@ -82,3 +82,21 @@ When the HSF organises [workshops](https://indico.cern.ch/category/7971/)
...
+
+- That the meeting is being recorded and will be made public;
```
Comment (eduardo-rodrigues) [inline what_are_WGs.md:95]:
```suggestion
- The meeting is being recorded and will be made public (most likely on Indico and/or YouTube);
```

**Review by eduardo-rodrigues — APPROVED** [review]
Very good idea!

File: what_are_WGs.md [hunk 4]
```diff
@@ -82,3 +82,21 @@ When the HSF organises [workshops](https://indico.cern.ch/category/7971/)
 working group conveners can give input to the organising team. If there
 will be a session dedicated to the activities of any working group then
 conveners will be asked to co-organise it.
+
+### Recording Meetings and Minutes
+
+In general recording *is appropriate* for HSF meetings, which are there to
```
Comment (eduardo-rodrigues) [inline what_are_WGs.md:88]:
```suggestion
In general, recording *is appropriate* for HSF meetings, which are there to
```

File: what_are_WGs.md [hunk 5]
```diff
@@ -82,3 +82,21 @@ When the HSF organises [workshops](https://indico.cern.ch/category/7971/)
...
+foster discussion and collaboration so *we do recommend it*. There are a lot of
```
Comment (eduardo-rodrigues) [inline what_are_WGs.md:89]:
```suggestion
foster discussion and collaboration, so *we do recommend it*. There are a lot of
```

**Comment by graeme-a-stewart** [comment]
Thanks for the comments @eduardo-rodrigues !

File: what_are_WGs.md [hunk 6]
```diff
@@ -85,14 +85,14 @@ conveners will be asked to co-organise it.
...
+foster discussion and collaboration, so *we do recommend it*. There are a lot of
 meetings and we have a worldwide community, so not everyone who is interested
-can join the meeting and a recording can help people to catch-up.
+can join a meeting and recordings can help people to catch-up.
 
-However, you need to make make it very clear with the speakers and to participants that:
+However, you need to make it very clear with the speakers and to participants that:
 
-- That the meeting is being recorded and will be made public;
+- The meeting is being recorded and will be made public;
```
Comment (hegner) [inline what_are_WGs.md:95]: IMHO the "that" belongs to the beginning of the line (removing it from above). Just because of the following bullet point

Comment (graeme-a-stewart) [reply what_are_WGs.md:95]: Yes, agreed - thanks.

**Comment by hegner** [comment]
@graeme-a-stewart - thanks!

**Pull request merged**